bundles_sort.go

```diff
@@ -8,9 +10,13 @@ func (b SortBundlesByName) Less(i, j int) bool { return b[i].Name < b[j].Name }
 
 type SortBundlesByVersion []Bundle
 
-func (b SortBundlesByVersion) Len() int           { return len(b) }
-func (b SortBundlesByVersion) Swap(i, j int)      { b[i], b[j] = b[j], b[i] }
-func (b SortBundlesByVersion) Less(i, j int) bool { return b[i].Version < b[j].Version }
+func (b SortBundlesByVersion) Len() int      { return len(b) }
+func (b SortBundlesByVersion) Swap(i, j int) { b[i], b[j] = b[j], b[i] }
+func (b SortBundlesByVersion) Less(i, j int) bool {
+	verA := semver.New(b[i].Version)
```

Comment:
Why rename from `i` to `A` here? `verI` and `verJ` would be easier to read for me atleast.

Reply:
Merged already, but feel free to make PR if you think this is important. I don't have any strong opinions here.